### PR TITLE
Setting `options` in findAll if `undefined`

### PR DESCRIPTION
**Pull request by mbad0la**
This is in reference to #56 

Notify : @gr2m

### CODE REVIEW

**Comment by gr2m** [comment]
@mbad0la can you add a find-all-test.js here: https://github.com/hoodiehq/hoodie-account-server-api/tree/master/test/unit/accounts and add two test cases, one with passing options, one without?

**Comment by mbad0la** [comment]
@gr2m my Travis build is failing upon the addition of the tests. Could you have a look at it and direct me on the corrections?

File: test/unit/accounts/find-all-test.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        get: simple.stub().resolveWith({
+          _id: 'org.couchdb.user:foo',
+          _rev: '1-234',
+          name: 'foo',
+          roles: ['id:user123'],
+          profile: {
+            name: 'bar'
+          }
+        }),
+        put: simple.stub().resolveWith({})
```
Comment (gr2m) [inline test/unit/accounts/find-all-test.js]: as these are unit tests, we need to mock all outside state and logic. Here you are mocking the PouchDB instance `state.db` and its `.get()` and `.put()` methods. The error that you currently get is `state.db.allDocs is not a function`. Which tells you that `findAllAccount()` calls `state.db.alldocs()` ([here](https://github.com/hoodiehq/hoodie-account-server-api/blob/2a5785502ffe6301e137dbc66e76757ee8cf8415/lib/accounts/find-all.js#L6)), so you need to mock its behaviour

Comment (mbad0la) [inline test/unit/accounts/find-all-test.js]: Got it. Thanks!

Comment (mbad0la) [inline test/unit/accounts/find-all-test.js]: @gr2m please review

**Pull request merged**